### DISCUSSION TITLE
1%20computer-principle/FPGA

### DISCUSSION THREAD

**Discussion by giscus** [kangkaisen/olap-performance#5] [Announcements]
# 1%20computer-principle/FPGA

资料 Data Processing with FPGAs on Modern Architectures (https://dl.acm.org/doi/pdf/10.1145/3555041.3589410);

https://perf.bcmeng.com/1%20computer-principle/FPGA.html

**Comment by Lhfcws** [comment]
都 introduce FPGA 了，感觉优化思路那里 offloading 也可以加一个小part。

**Reply by kangkaisen** [reply]
好的，感谢反馈
